src/jit/codegenarmarch.cpp

```diff
@@ -1589,7 +1589,7 @@ void CodeGen::genCodeForLclFld(GenTreeLclFld* tree)
     unsigned varNum = tree->gtLclNum;
     assert(varNum < compiler->lvaCount);
 
-    if (varTypeIsFloating(targetType))
+    if (varTypeIsFloating(targetType) || varTypeIsSIMD(targetType))
```

Comment:
It's worth noting (though not sure it's worth a comment) that `varTypeIsSIMD` returns `false` if `FEATURE_SIMD` is not enabled.